.ci/run_rhub_solaris_checks.R

```diff
@@ -0,0 +1,87 @@
+args <- commandArgs(
```

Comment:
@jameslamb Please forgive me for this R code! 🙈 
Feel free to push any changes directly to the branch 😃 .

Reply:
it's very good, don't worry!